frontend/src/scenes/events/EventJSON.tsx

```diff
@@ -20,5 +20,15 @@ export function EventJSON(props: { event: Record<string, any> }): JSX.Element {
         ...otherProps,
     }
 
-    return <CodeSnippet language={Language.JSON}>{JSON.stringify(newEvent, null, 4)}</CodeSnippet>
+    return (
+        <CodeSnippet
+            codeStyle={{
+                padding: '1rem',
+                margin: '0.5rem',
+            }}
+            language={Language.JSON}
+        >
+            {JSON.stringify(newEvent, null, 4)}
+        </CodeSnippet>
+    )
```

Comment:
please ignore ugly one-off styles in EventJSON and EventElements. These components should be refactored eventually but not here.